data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_12.xml.ep

```diff
@@ -109,12 +109,8 @@
         <bootproto>dhcp</bootproto>
         <bridge>yes</bridge>
         <bridge_forwarddelay>15</bridge_forwarddelay>
-        <!-- Use specified interface instead of a list to avoid ip lease inconsistency. Please refer to poo#135641. -->
-        % if ($check_var->('SUT_NETDEVICE', 'eth1') or $check_var->('SUT_NETDEVICE', 'em1')) {
-        <bridge_ports><%= $get_var->('SUT_NETDEVICE') %></bridge_ports>
-        % } else {
-        <bridge_ports>eth0 eth1 em1 em2</bridge_ports>
-	% }
+        <!-- Use specified interface instead of a list to avoid bonding issues which act as router and created illegal duplicate traffic for gateway of its subnet. -->
+        <bridge_ports><%= $get_var->('BRIDGE_PORT', 'eth0') %></bridge_ports>
```

Comment:
If BRIDGE_PORT use it, otherwise use SUT_NETDEVICE ?

Reply:
No, the bridge is default to `eth0` if there is no `BRIDGE_PORT`.

`<bridge_ports><%= $get_var->('BRIDGE_PORT', 'eth0') %></bridge_ports>`

Reply:
Why not use SUT_NETDEVICE if BRIDGE_PORT is not present ?

Reply:
`SUT_NETDEVICE` is an optional setting, and its value might be a MAC instead of an interface name.

Reply:
Yes. When it is MAC, you use BRIDGE_PORT.  When BRIDGE_PORT is not present, use SUT_NETDEVICE instead. Right ?

Reply:
?? it is as complex as the way without the new setting. Since we have introduced a new setting,  we need not implement it so complicatedly, this logic only saves one `BRIDGE_PORT` on one machine `gonzo`. If you realy bother with two more settings, there is no reason that we introduced a new one.